runelite-client/src/main/java/net/runelite/client/plugins/wiki/config/WikiLeftClickOption.java

```diff
@@ -0,0 +1,20 @@
+package net.runelite.client.plugins.wiki.config;
```

Comment:
License header.

Also, this does not need to be in a config directory, it can be in the same dir as the rest of the plugin files.

Reply:
I copied the structure from `grounditems` and `wintertodt`. Should I change it?

Reply:
I suppose it can stay where it is. It doesn't really make much of a difference, but being in a separate package requires import statements in other plugin files to access it.